plugins/content-length.rst

```diff
@@ -1,4 +1,35 @@
 Content-Length Plugin
 =====================
 
-TODO
+The `ContentLengthPlugin` sets the correct `Content-Length` header value based on the size of the body stream of the
```

Comment:
in .rst, you need double backticks for code, single backticks won't work. (needs to be fixed in all places, won't repeat it everywhere ,-) )